webapp/src/modules/authorization/sagas.js

```diff
@@ -77,8 +81,17 @@ function* handleAllowTokenRequest(action) {
     const contractToApprove = eth.getContract(contractName)
     const tokenContract = eth.getContract(tokenContractName)
 
+    // AUCTION
+    let options
+    const pathname = yield select(getPathname)
+    if (pathname === locations.auction()) {
+      const { gasPriceLimit } = yield select(getParams)
+      const gasPrice = gasPriceLimit * 1000000000
```

Comment:
Not sure about this one, but putting it out there.

I find the `* 1000000000` and `/ 1000000000` a bit confusing. `/auction/sagas` is doing it to (with `const ONE_BILLION = 1000000000`).

I don't want to add an utils function on `auction/utils` because `gasPrice` is not auction only. So right know I'd advise we keep an eye on this, specially if we decide to start hardcoding gas values on every tx

Reply:
👍